lib/bugsnex/plug.ex

```diff
@@ -75,4 +77,21 @@ defmodule Bugsnex.Plug do
     {:ok, hostname} = :inet.gethostname
     to_string(hostname)
   end
+
+  defp filter_parameters(params) do
+    do_filter(params, Application.get_env(:bugsnex, :filter_params, @default_filter_params))
+  end
+
+  defp do_filter(%{__struct__: mod} = struct, _params) when is_atom(mod), do: struct
```

Comment:
This avoids altering structs, which means that if somehow a struct ends up in the params (which it generally shouldn't) we wouldn't sanitize its keys. But it's keys would be atoms. And it should never happen. We are probably fine. Just thinking out loud.